datastore/tests/unit/test_helpers.py

```diff
@@ -135,6 +135,27 @@ def test_mismatched_value_indexed(self):
         with self.assertRaises(ValueError):
             self._call_fut(entity_pb)
 
+    def test_index_mismatch_ignores_empty_list(self):
+        from google.cloud.proto.datastore.v1 import entity_pb2
+        from google.cloud.datastore.helpers import _new_value_pb
+
+        _PROJECT = 'PROJECT'
+        _KIND = 'KIND'
+        _ID = 1234
```

Comment:
Just for fun: can you make this "normal" names for locals, i.e. `project`, `kind` and `id_`?